elm.json

```diff
@@ -11,10 +11,10 @@
     "dependencies": {
         "elm/core": "1.0.5 <= v < 2.0.0",
         "elm/project-metadata-utils": "1.0.2 <= v < 2.0.0",
-        "jfmengels/elm-review": "2.4.0 <= v < 3.0.0",
+        "jfmengels/elm-review": "2.11.0 <= v < 3.0.0",
```

Comment:
Looks like the minimum version is being advanced since we are now using `Rule.providesFixesForProjectRule` which was added in `2.10.0`:

https://github.com/jfmengels/elm-review/blob/master/CHANGELOG.md#2100---2022-11-08

Should we relax this to `2.10.0` instead of requiring at least `2.11.0`? Or, are there other changes in `2.11.0` such that any user of this rule needs be using at least `elm-review@2.11.0`?

Reply:
In practice it doesn't matter much (I tend to often advance the version for my packages) but you're right, let's open it up to 2.10.0 :+1: